docs/light/api.md

```diff
@@ -403,21 +393,30 @@ Returns on error:
 }
 ```
 
-### /bank/create_transfer - POST
+### POST /bank/transfers
 
-url: /bank/create_transfer
+url: `/bank/transfers`
 
 Functionality: Create a transfer in the bank module.
 
-Parameters:
```

Comment:
@jackzampolin Should we get rid of the parameter table ? I think they are useful and combined with the Post body would make it more clear for the end user